tools/cldr-code/src/main/java/org/unicode/cldr/util/Level.java

```diff
@@ -12,14 +12,14 @@
  *
  */
 public enum Level {
-    UNDETERMINED(0, "none", 0),
-    CORE(10, "G4", 100),
-    BASIC(40, "G3", 80),
-    MODERATE(60, "G2", 70),
-    MODERN(80, "G1", 50),
-    COMPREHENSIVE(100, "G0", 2);
+    UNDETERMINED(0, "none", 0, "�"),
```

Comment:
intentionally U+FFFD? Would something else be better?

Reply:
It should never show up, so that's just a visual reminder!